Users/service.js

```diff
@@ -1,51 +1,63 @@
 var response = require('../Objects/response.js');
 
-var createUser = function (json) {
-    // Creates a user
-
-    // TODO: Create user and remove console.log
-    console.log(json);
+var createUser = (json) => {
+	// Create Student
+	if (json.role == 'Student') {
+		var Student = new StudentModel({
+			approvalStatus: json.approvalStatus,
```

Comment:
Should ```approvalStatus``` be accepted for this method?  This way, a student could potentially approve themselves when they create their account.